.gitignore

```diff
@@ -1,4 +1,5 @@
 .*
+!.github
```

Comment:
🤦🏼

Reply:
Agreed. We had a bunch of hidden folders to exclude, but went too far in https://github.com/mozilla/pontoon/pull/3401